virtcontainers/experimental/experimental.go

```diff
@@ -0,0 +1,32 @@
+// Copyright (c) 2019 Huawei Corporation
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+package experimental
+
+import (
+	"fmt"
+)
+
+// Feature to be experimental
+type Feature string
```

Comment:
We could do something a little more structured like this:

```go
type Feature struct {
    Name        string
    Description string
    Tags        []string
}
```

So when that gets printed using `%+v`, we get something like:

```go
{Name:unicorn Description:spreads sunshine and love and makes magic happen Tags:[density performance security]}
```

That might seem like overkill, but we do have to document these features somewhere, so why not document them where they can be seen?! :smile:

Reply:
@jodh-intel append a new commit, PTAL. Thanks!